packages/longhorn/package.yaml

```
@@ -1,7 +1,7 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: 37f3263c5f7d3361550521c9e36968b5f3d9b7b4
-version: 100.0.0
+commit: f629c0352211bb04eb0e7a97472c779078e971c3
+version: 101.0.0
```

```suggestion
version: 100.0.1
```

We only bump major / minor version during Rancher minor releases